static/js/theme-map/Maps/Providers/Mapbox.js

```diff
@@ -175,22 +183,38 @@ const yextAPIKey = 'pk.eyJ1IjoieWV4dCIsImEiOiJqNzVybUhnIn0.hTOO5A1yqfpN42-_z_GuL
  * @see {MapProvider~loadFunction}
  */
 function load(resolve, reject, apiKey, {
-  version = 'v1.6.1'
+  version = 'v1.13.0'
 }) {
   const baseUrl = `https://api.mapbox.com/mapbox-gl-js/${version}/mapbox-gl`;
 
   const mapStyle = document.createElement('link');
   mapStyle.rel = 'stylesheet';
   mapStyle.href = baseUrl + '.css';
 
+  const mapLanguageScript = document.createElement('script');
+  mapLanguageScript.src = 'https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-language/v0.10.1/mapbox-gl-language.js';
```

Comment:
would it be possible to import this as an npm module and import it with a dynamic import?

Reply:
actually ignore this, I think if they're hosting it already there's not much point using a dynamic import